client/src/contexts/userContext.tsx

```diff
@@ -36,6 +39,8 @@ const reducer = (state: UserState, action: UserAction): UserState => {
       };
     case 'LOGOUT':
       return {};
+    case 'UPDATE_USER':
+      return { ...state, ...action.payload };
```

Comment:
작업이 어떤 의미인지 모호하네요... payload에서 innerText만 받고 
아래와 같이 수정하는 건 어떨까요
```suggestion
     return {...state, currentProject : action.payload }
```